flash/video/classification/input_transform.py

```diff
@@ -25,11 +25,15 @@
 
 if _PYTORCHVIDEO_AVAILABLE:
     from pytorchvideo.transforms import UniformTemporalSubsample
-    from torchvision.transforms import CenterCrop, Compose, RandomCrop
+    from torchvision.transforms import CenterCrop, Compose, Lambda, RandomCrop
 else:
     ClipSampler, LabeledVideoDataset, EncodedVideo, ApplyTransformToKey = None, None, None, None
 
 
+def lambda_tmp(x: torch.Tensor) -> torch.Tensor:
```

Comment:
Perhaps `normalize` would be better:
```suggestion
def normalize(x: torch.Tensor) -> torch.Tensor:
```